app/src/main/java/ltd/evilcorp/atox/tox/ToxIdValidator.kt

```diff
@@ -5,11 +5,13 @@ class ToxIdValidator {
         NO_ERROR,
         INCORRECT_LENGTH,
         INVALID_CHECKSUM,
+        NOT_HEX,
     }
 
     companion object {
         fun validate(toxID: ToxID) = when {
             toxID.string().length != 76 -> Result.INCORRECT_LENGTH
+            !toxID.string().matches(Regex("[0-9A-Fa-f]*")) -> Result.NOT_HEX
```

Comment:
Did you consider accepting and stripping `\s`?

Reply:
I did, but I decided that the user can strip their own whitespace until someone opens a PR with that. 👼